gobblin-utility/src/main/java/gobblin/util/ClassUtils.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.util;
+
+import java.util.List;
+
+
+/**
+ * Utility class to manage class lifecycle.
+ */
+public class ClassUtils {
```

Comment:
This class is unnecessary. Look at https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/reflect/ConstructorUtils.html#invokeConstructor(java.lang.Class, java.lang.Object[])